src/utils.ts

```diff
@@ -284,6 +290,61 @@ export async function getBrowserPath(config: Partial<IUserConfig> = {}): Promise
     }
 }
 
+/**
+ * Gets a supported debug config and updates the status of the launch.json file associated with the current workspace
+ * @returns {vscode.DebugConfiguration | null}
+ */
+ export function getLaunchJson(): vscode.DebugConfiguration | null {
+    // Check if there is a folder open
+    if (!vscode.workspace.workspaceFolders) {
+        vscode.commands.executeCommand('setContext', 'launchJsonStatus', "None");
+        return null;
+    }
+
+    // Check if there's a launch.json file
+    const workspaceUri = vscode.workspace.workspaceFolders[0].uri;
+    const filePath = `${workspaceUri.fsPath}/.vscode/launch.json`;
+    if (fse.pathExistsSync(filePath)) {
+        // Check if there is a supported debug config
+        const configs = vscode.workspace.getConfiguration('launch', workspaceUri).get('configurations') as vscode.DebugConfiguration[];
+        for (const config of configs) {
+            if (config.type === 'vscode-edge-devtools.debug' || config.type === 'msedge' || config.type === 'edge') {
+                vscode.commands.executeCommand('setContext', 'launchJsonStatus', "Supported");
+                return config;
+            }
+        }
+        vscode.commands.executeCommand('setContext', 'launchJsonStatus', "Unsupported");
+        return null;
+    } else {
+        vscode.commands.executeCommand('setContext', 'launchJsonStatus', "None");
+        return null;
+    }
+}
+
+/**
+ * Add a template for a supported debug configuration to launch.json
+ * @returns {void}
+ */
+export function configureLaunchJson(): void {
+    if (!vscode.workspace.workspaceFolders)
+        return;
+    
+    // Create ./.vscode/launch.json if it doesn't already exist
+    const workspaceUri = vscode.workspace.workspaceFolders[0].uri;
+    fse.ensureFileSync(`${workspaceUri.fsPath}/.vscode/launch.json`);
+
+    // Append a supported debug config to their list of configurations
+    const launchJson = vscode.workspace.getConfiguration('launch', workspaceUri);
+    let configs = launchJson.get('configurations') as vscode.DebugConfiguration[];
+    let configWithInstruction = {...providedDebugConfig};
+    configWithInstruction.url += ' **Replace with your website url before launching**';

```

Comment:
It might be a cleaner experience if we instead default to the new startpage in the generated launch.json, and leave a comment after the url that contains this text. That way, the generated launch.json is valid without any edits?

Reply:
That would depend on #350 . Or we could default to the localhost link for now and I could update my PR if this lands first